Gemfile

```diff
@@ -25,6 +25,14 @@ group(:development, :test) do
   gem 'cheffish', "~> 1.3", "!= 1.3.1"
 
   gem 'ruby-shadow', :platforms => :ruby unless RUBY_PLATFORM.downcase.match(/(aix|cygwin)/)
+
+  # For external tests
+  gem 'chefspec', github: 'jkeiser/chefspec', branch: 'jk/chefspec-12.5'
+  gem 'chef-sugar'
+  gem 'poise', github: 'poise/poise', branch: 'deeecb890a6a0bc2037dfb09ce0fd0a8931519aa'
```

Comment:
@coderanger the munger creates a new Gemfile in a temp location, so ../specs will not do what one expects.

I *could* create a new Gemfile in the gem directory itself, though. Probably would be fine.